src/pages/index.jsx

```diff
@@ -45,56 +47,58 @@ const App = () => {
       <div className="home">
         <section className="header">
```

Comment:
I recommend splitting up the sections into their own components into their own "sections" directory